packages/strapi-plugin-upload/admin/src/icons/Carret/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Carret = () => {
```

Comment:
I think this component already exists in the content manager can you make it work for both? Or maybe add it to **Buffet.js**